src/polarization.jl

```diff
@@ -0,0 +1,48 @@
+#################
+# polarization vectors
+# 
+# In this file, we provide an extension of the Polarizations of Photons from
+# QEDbase to define polarization states of background fields
+################
+#
+# TODO:
+#   * write doc strings
```

Comment:
Do you want to add the doc-strings with another PR, or with this one? (I think it is almost done anyways.)

Reply:
You are right! cb77bf0 removes the TODO.